pipelines/build/common/openjdk_build_pipeline.groovy

```diff
@@ -292,6 +292,40 @@ class Build {
 
         return jdkRepo
     }
+
+    private String allocateEBCnodesForTest(testType) {
+        def test_target = buildConfig.TARGET_OS + '_' + buildConfig.ARCHITECTURE
+        // All targets are not supported via EBC yet. We need to test that target and add to default.json later on to make it enable
+        if ( ! DEFAULTS_JSON['testDetails']['ebcEnabledTargets'][test_target].find { it == testType } ) {
+            context.println "EBC does not support for $test_target->$testType yet!"
+            return ''
+        }
+
+        // context.library 'jenkins-ci-websphere' 
+        def group_label_UUID = 'Adopt-semeru_test_machine_' + UUID.randomUUID().toString()
+        def num_machines = '0'
+        def test_index = DEFAULTS_JSON['testDetails']['defaultDynamicParas']['testLists'].indexOf(testType)
+        num_machines = DEFAULTS_JSON['testDetails']['defaultDynamicParas']['numMachines'].get(test_index)
+        def nodes_ebc_args = new Object[num_machines]
+
+        if (num_machines == '0' ) {
+            num_machines = '1'
+        }
+
+        def jobName = '/EBC_Create_Node'
```

Comment:
Do we need the leading `/` here?

Reply:
Yes. The job is created in /, removing that means it is a relative path and job resides beside the pipeline. As we use this module almost everywhere it is not possible to create same EBC job in every relative path.

Reply:
I've never used `/` but I'm indifferent one way or the other.